sljit_src/sljitConfig.h

```diff
@@ -96,7 +96,7 @@ extern "C" {
 
 /* Executable code allocation:
    If SLJIT_EXECUTABLE_ALLOCATOR is not defined, the application should
-   define SLJIT_MALLOC_EXEC, SLJIT_FREE_EXEC, and SLJIT_EXEC_OFFSET. */
+   define SLJIT_MALLOC_EXEC and SLJIT_FREE_EXEC. */
```

Comment:
We should describe that there is an optional `SLJIT_EXEC_OFFSET` macro, since it might be needed in some cases.

Reply:
the same applies to SLJIT_UPDATE_WX_FLAGS, but since it was not mentioned and had already a default fallback assumed these only made the formal interface cleaner.